public/data/EddieHubCommunity.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "EddieHubCommunity",
+  "bio": "Community of inclusive Open Source people - Collaboration 1st, Code 2nd!",
+  "avatar": "https://github.com/EddieHubCommunity.png",
+  "links": [
+    {
+      "name": "Visit Website",
+      "url": "https://eddiehub.org",
```

The current link doesn't work 👀 no idea why.
```suggestion
      "url": "https://www.eddiehub.org/",
```

---

I think www. Redirects to the without www. Version

---

I think it is the https

```suggestion
      "url": "http://eddiehub.org",
```